web/src/environments/environment.ts

```diff
@@ -7,12 +7,18 @@ export const environment = {
   version: 'x.x.x',
   // Add dev firebase credentials
   firebase: {
-    apiKey: '',
-    authDomain: '',
-    databaseURL: '',
-    projectId: '',
-    storageBucket: '',
-    messagingSenderId: ''
+    // apiKey: '',
+    // authDomain: '',
+    // databaseURL: '',
+    // projectId: '',
+    // storageBucket: '',
+    // messagingSenderId: ''
+    apiKey: 'AIzaSyA_YnmL2hlmjNi1J6i47PJwR6yOwsuSrdk',
+    authDomain: 'tribedashboard-dev.firebaseapp.com',
+    databaseURL: 'https://tribedashboard-dev.firebaseio.com',
+    projectId: 'tribedashboard-dev',
+    storageBucket: 'tribedashboard-dev.appspot.com',
+    messagingSenderId: '180845259287'
```

Comment:
I think this got committed by accident again

Reply:
Apologies Will remove it